edge-agent/docker/linux/arm64v8/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.6.10-linux-arm64v8
+ARG base_tag=1.0.6.11-linux-arm64v8
```

Comment:
Why did you overwrote my commit to update the base image with your own?!?! 
I knew it is error prone and that's why I volunteered to do it. 
You forgot to update  `azureiotedge-agent-base:1.0.6.10-linux-arm32v7` here.
Now... we just accidentally push `azureiotedge-agent-base:1.0.6.10-linux-arm32v7` which is a legacy base image because it's base off of this PR (which never merged to be into the "managed" branch. 
Either close the PR or merge it. NO FERMENATION DUNGEON FOR BASE IMAGE PLEASE!

Reply:
As a follow up of this fiesta, the follow steps need to be taken:
1. Publish the base image for `azureiotedge-agent-base:1.0.6.11-linux-arm32v7` (once the base image is fixed in this PR).
2. Republish the base image for `azureiotedge-agent-base:1.0.6.10-linux-arm32v7` with the achieved commit in the `release/1.1` branch.